java-desktop/src/main/java/com/couchbase/todo/controller/TaskListController.java

```diff
@@ -219,7 +220,6 @@ private void updateTaskName(@NotNull String id, @NotNull String name) {
         // TODO: Execute on non-ui thread
         Document doc = DB.get().getDocument(id);
         if (doc == null) { return; }
-
         MutableDocument mDoc = doc.toMutable();
```

Comment:
awww.... leave that pretty little empty line!  It makes it so much easier to read.... ;-P